CHANGELOG.md

```diff
@@ -17,6 +17,9 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#2050](https://github.com/realm/SwiftLint/issues/2050)
 
+* Fix linux crash on sources with surrogate pair emojis as variable names
```

Comment:
requires period and two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes